networking/private-cloud-function-from-onprem/variables.tf

```diff
@@ -73,8 +26,34 @@ variable "ip_ranges" {
   }
 }
 
+variable "name" {
+  description = "Name used for new resources."
+  type        = string
+  default     = "psc-onprem"
```

Comment:
I suggest changing the default name, otherwise with the current value you would end up with VPCs named `psc-onprem-hub` and `psc-onprem-onprem` which I find a bit confusing